router/server.go

```diff
@@ -61,6 +61,8 @@ func NewRouter(ctx context.Context, h models.HandlerInterface, port int) *Router
 		Methods("GET")
 	gMux.Handle("/api/perf/result", h.ProviderMiddleware(h.AuthMiddleware(h.SessionInjectorMiddleware(h.GetResultHandler)))).
 		Methods("GET")
+	gMux.Handle("/api/perf/mesh/list", h.ProviderMiddleware(h.AuthMiddleware(h.SessionInjectorMiddleware(h.GetSMPServiceMeshes)))).
```

Comment:
Let's give holistic consideration to the endpoints that we are exposing. A list of possible service meshes is interesting under a number of uses, not strictly in context of performance.

We need to get together of set of API docs (@anirudhjain75 or @chandrashritii  do you have this?) - or simply a set of endpoints into a categorized list.

Reply:
@leecalcote does `/api/mesh/list` look better?

Reply:
@utkarsh-pro, yes, or simply a `/api/mesh/` a GET on this resource should return this list. Later, it should also return information on installed and actively running service meshes.